shipyard-controller/handler/sequence_migrator.go

```diff
@@ -56,7 +57,6 @@ func (sm *SequenceMigrator) migrateSequencesOfProject(projectName string, wg *sy
 	pageSize := int64(50)
 
 	for {
```

Comment:
Can you please explain why this does not produce an endless loop? 
I cannot see how the `NextPageKey` is used?

Reply:
right, I have fixed that now

Reply:
Thank you!